gateway-ha/src/test/java/io/trino/gateway/ha/router/TestStochasticRoutingManager.java

```diff
@@ -66,7 +71,9 @@ public void addMockBackends()
             haRoutingManager.updateBackEndHealth(backend, false);
         }
 
-        assertThat(haRoutingManager.provideBackendForRoutingGroup(groupName, ""))
+        headers.add(USER_HEADER, "");
```

Comment:
Could you add another test for not setting `USER_HEADER` at all?

Reply:
of course :)